src/binding.c

```diff
@@ -11,6 +12,13 @@
 #include "pafish/wine.h"
 #include "pafish/sandboxie.h"
 
+#define MAX_SAMPLE_SIZE ((1024 * 1024))
+#define MAX_INTERVAL_MILLISECONDS (100)
```

Comment:
this is a really nice addition. Without this check, this function might impose a DDOS vunerability.

pafish uses [500ms](https://github.com/a0rtega/pafish/blob/6c1fabdf8a4c4b61c37ef41e17c223611b15bbde/pafish/cpu.c#L89) as interval.

I think we should atleast allow up to that value.

Reply:
sure

Reply:
on second note, I'm not so sure anymore. While running the test with maxed sample sizes I had to kill the job after I realized it was running for like 30 minutes. We should really look to reduce this waaay back IMNSHO